Fluid/TemplateOptions.cs

```diff
@@ -120,7 +120,9 @@ public TemplateOptions()
             Filters.WithArrayFilters()
                 .WithStringFilters()
                 .WithNumberFilters()
-                .WithMiscFilters();
+                .WithMiscFilters()
+                .WithColorFilters();

```

Comment:
Nice catch, seems I forgot to add it before

Reply:
I think it's actually on purpose because these are not part of the liquid spec and such are usually only opt in. What could be missing is a doc section listing all the With methods and special filters.

Reply:
https://shopify.dev/docs/api/liquid/filters/color_to_hex

Reply:
This is Shopify specific. Like we have custom filters in Orchard. True Liquid reference is taken from https://github.com/Shopify/liquid. Anything that is in this repo and documented in Shopify is fine to use though.

Reply:
Unfortunately, we did not mention how to create a custom filter(s). We could improve the docs

Reply:
I got it, I will revert the latest commit, then write clear docs about it